govc/USAGE.md

```diff
@@ -4271,6 +4302,42 @@ Options:
   -l=false               Long listing format
 ```
 
+## namespace.create
+
+```
+Usage: govc namespace.create [OPTIONS]
+
+Creates a new namespace on a supervisor cluster. 
+
+Examples:
+  govc namespace.create -cluster=test-cluster -namespace=test-namespace
+  govc namespace.create -cluster=test-cluster -namespace=test-namespace -content-libraries=lib1
+  govc namespace.create -cluster=test-cluster -namespace=test-namespace -content-libraries=lib1,lib2
+  govc namespace.create -cluster=test-cluster -namespace=test-namespace -vm-classes=class1
+  govc namespace.create -cluster=test-cluster -namespace=test-namespace -vm-classes=class1,class2
+  govc namespace.create -cluster=test-cluster -namespace=test-namespace -content-libraries=lib1,lib2 -vm-classes=class1,class2
```

Comment:
Don't we use the content library UUID and not the names (since you can have different CL with the same name)

Reply:
We use the UUID, yes, that's a good point.
I'll modify update the docs